Gruntfile.js

```diff
@@ -46,7 +49,7 @@ module.exports = function(grunt) {
           files: [
             {pattern: 'dist/jose-jwe.js', watching: false, included: false},
             {pattern: 'test/qunit-promises.js', watching: false, included: false},
-            'test/jose-jwe-test.html'
+            'test/jose-jws-test.html'
```

Comment:
we should keep jose-jwe-test.html and add jose-jws-test.html as an additional test.